app/kumactl/cmd/apply/apply.go

```diff
@@ -2,6 +2,8 @@ package apply
 
 import (
 	"context"
+	"github.com/Kong/kuma/app/kumactl/pkg/output"
+	"github.com/Kong/kuma/app/kumactl/pkg/output/printers"
```

Comment:
nitpicking: we sort imports like this
```
go std imports (like `context`)
new line
external imports (like `pkg/errors`)
new line
kuma imports (like `github.com/Kong/kuma/app/kumactl/pkg/output/printers`)
```

You'll find many files in the project that does not stick to this convention, but lets try to keep it this way.

Reply:
Fixed. Do we want to automate this process eventually? For example use [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports)?

Reply:
Yes please! If this can be automated, let's do it. Not in this PR though. Create a separate PR or create a Github issue, so we won't forget.